ec2utils/ec2publishimg/ec2publishimg

```diff
@@ -25,6 +25,7 @@ import sys
 
 import ec2utils.ec2utilsutils as utils
 import ec2utils.ec2publishimg as ec2pubimg
+from ec2utils.ec2publishimg_utils import validate_accounts
```

Comment:
Nope, not doing this. Either validate_account_numbers moves directly into this code or into utilsutils